src/api/parity.rs

```diff
@@ -31,6 +32,23 @@ impl<T: Transport> Parity<T> {
 
         CallFuture::new(self.transport.execute("parity_call", vec![reqs]))
     }
+
+    /// Get pending transactions
+    /// Blocked by https://github.com/openethereum/openethereum/issues/159
+    pub fn pending_transactions(
+        &self,
+        limit: Option<usize>,
+        filter: Option<ParityPendingTransactionFilter>,
+    ) -> CallFuture<Vec<Transaction>, T::Out> {
+        let params = match (limit, filter) {
+            (Some(l), Some(f)) => vec![l.into(), helpers::serialize(&f)],
+            (Some(l), _) => vec![l.into()],
+            (_, Some(f)) => vec![Value::Null, helpers::serialize(&f)],
+            _ => vec![],
+        };
```

Comment:
```suggestion
     let limit = limit.map(Value::from);
     let filter = filter.as_ref().map(helpers::serialize);
     let params = match (limit, filter) {
            (l, Some(f)) => vec![l.unwrap_or(Value::Null), f],
            (Some(l), None) => vec![l],
            _ => vec![],
        };
```
This avoids duplication.